index.html

```diff
@@ -227,6 +231,29 @@ <h1 class="about-title">About <br>Myself</h1>
     </div>
 
   </section>
+
+  <footer class="contact-form">
+
+    <div class="form-bg"></div>
+
+    <div class="from-descript">
+
+      <h2 class="form-title">Contact me</h2>
+      <div class="contact-info">
+        <p>If you have an application you are interested in developing, a feature that you need build or a project that needs coding. I'd love to help with.</p>
+      </div>
+      
+      <form class="form-action" action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- Well done using the `post` method on the form, but you are required to sign up on the [formspree service](https://formspree.io/html) and integrate your form using the form_id